src/components/DevicesView.jsx

```diff
@@ -8,18 +8,28 @@ import { translate } from 'cozy-ui/react/helpers/i18n'
 import Loading from './Loading'
 import Empty from './Empty'
 
+import DevicesModaleRevokeView from './DevicesModaleRevokeView'
+
 // for the icon, we show a phone if it's a phone, and a laptop in all other cases
 const getDeviceKindClass = kind => kind === 'mobile' ? styles['set-device-phone'] : styles['set-device-laptop']
 
-const DevicesView = ({ t, f, isFetching, devices }) => (
+const DevicesView = ({ t, f, isFetching, devices, openDeviceRevokeModale, deviceToRevoke, onDeviceModaleRevoke, onDeviceModaleRevokeClose, devicePerformRevoke }) => (
   <div className={devicesStyles['devices-view']}>
     <h2>{t('DevicesView.title')}</h2>
     { isFetching && <Loading /> }
     { !isFetching && devices.length === 0 && <Empty emptyType='devices' />}
-    { devices.length > 0 && (
+    { !isFetching && devices.length > 0 && (
```

Comment:
Ah, I see!
I think that `isFetching` is used to indicate when the initial data is being fetched, and not so much for subsequent AJAX calls. The [info and password updating](https://github.com/cozy/cozy-settings/blob/master/src/actions/index.js#L37) for example don't change that flag.
Is it really useful to change that `fetching` flag in this case? If not I would suggest to remove it.